v22.2/aws-dms.md

```diff
@@ -176,6 +190,20 @@ The `BatchApplyEnabled` setting can improve replication performance and is recom
 
     Try selecting **Full LOB mode** in your [task settings](#step-2-2-task-settings). If this does not resolve the error, select **Limited LOB mode** and gradually increase the **Maximum LOB size** until the error goes away. For more information about LOB (large binary object) modes, see the [AWS documentation](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.LOBSupport.html).
 
+- If you encounter an `ABORT_REASON_CLIENT_REJECT` error when migrating an especially large table, and are running v22.2.4 or later, set the following session variables using [`ALTER ROLE ... SET {session variable}`](alter-role.html#set-default-session-variable-values-for-a-role):
```

Comment:
Replace this with TransactionProtoWithRefreshError

Reply:
@otan I take it you mean `TransactionRetryWithProtoRefreshError` and used this, but let me know if I'm wrong.